src/util/command_line_handling.cpp

```diff
@@ -816,6 +816,14 @@ namespace hpx { namespace util
                  vm.count("hpx:ignore-batch-env")) == 0) &&
             !use_process_mask_;
 
+#if defined(__APPLE__)
+        std::cerr
```

Comment:
Are you printing the warning message even if the `use_process_mask` option has not been used?

Reply:
Also not intentional. Thanks!